R/idmc_transform_daily.R

```diff
@@ -16,7 +32,12 @@
 #'   idmc_transform_daily()
 #'
 #' @export
-idmc_transform_daily <- function(df) {
+idmc_transform_daily <- function(
+    df,
+    min_date = min(as.Date("2018-01-01")),
+    max_date = Sys.Date(),
+    filter_min_date = TRUE
+  ) {
   # date columns to transform
   s_col_ <- "displacement_start_date"
```

Comment:
curious how you use`obj_name_` suffix in your code so that I can adopt

Reply:
So, this was a weird one. Basically, the variable names were so long, it was extremely messy to continually type them into the code, and thus I wanted to refer to them in the string format with `[[obj_name_]]` rather than `$obj_name`. The inclusion of the additional `_` at the end as a suffix, or sometimes as a prefix, was simply to denote that these are temporary objects that have no use further on. Often it's used even broader, where in Python a single leading underscore denotes a variable used solely within that class. Because of R's issue with object names with leading underscores, I used a tailing one here.

I haven't used really in a systematic way, cause technically I could even be naming interim data frames within the function in that manner. But in this case, because I was making them for a lot of us at that moment and not much later, I put that `_` there to simply denote that. We might think about how we use it in the future, but might not be necessary. Ideally I would simplify these functions further where arguments within the function are clear and it's short enough to clear identify what is being exported.